package.json

```diff
@@ -9,8 +9,7 @@
   },
   "main": "src/lightning.mjs",
   "scripts": {
-    "postinstall": "rollup -c",
-    "build": "rollup -c"
+    "postinstall": "rollup -c"
```

Comment:
Why are you removing the build task?  As a contributor on the project I need to be able to verify that the project will build after I make changes - it is very common to have run `npm run build` when doing active development on the project.